core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -150,8 +150,9 @@ private[spark] class AppStatusStore(
       Utils.tryWithResource(
         store.view(classOf[TaskDataWrapper])
           .parent(stageKey)
-          .index(TaskIndexNames.EXEC_RUN_TIME)
-          .first(0L)
+          .index(TaskIndexNames.STATUS)
+          .first("SUCCESS")
+          .last("SUCCESS")
```

Comment:
Doesn't this count all tasks between the first and last successful one, rather than count just successful ones?

Reply:
I see, thank you.